### ISSUE TITLE
aws_synthetics: The timeout from the RunConfig was missed so there is no way to change it

### ISSUE THREAD

**Issue by ignaciolarranaga** [aws/aws-cdk#28152]
### Describe the feature

Add the timeout, here is the reference:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-synthetics-canary-runconfig.html

As you can see here https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_synthetics.Canary.html it is not there.

### Use Case

Se the timeout

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### CDK version used

2.110.0

### Environment details (OS name and version, etc.)

NA

**Comment by pahud** [comment]
Yes we can expose that in the construct prop and before that we can always use [escape hatches](https://docs.aws.amazon.com/cdk/v2/guide/cfn_layer.html) to addPropertyOverride on it.

**Comment by mazyu36** [comment]
Hi.
This issue has been addressed in #30556  and released in [v2.151.0](https://github.com/aws/aws-cdk/releases/tag/v2.151.0).
I apologize that it wasn't automatically closed, possibly due to an error in my settings.
